### ISSUE TITLE
Queries on Object Navigation.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5374]
**Reported by iamjett27 on 2015-09-20 00:38**
Hi Friends! We are just a little bit confused on how to use these object navigations especially the following commands below. 
1. NVDA+numpad8 - Move to containing object
2. NVDA+numpad4 - Move to previous object
3. NVDA+numpad6 - Move to next object
4. NVDA+numpad2 - Move to first contained object
5. NVDA+numpadMinus - Move to focus object
6. NVDA+numpadEnter - Activate current navigator object

May we know also if where can we use these following commands? Please help us enlighten us regarding this matter. We hope that you can help us. Thank you and have a great day.

**Comment by bhavyashah** [comment]
NVDA support related questions should be sent either to the NVDA Users International forum (Groups.io e-mail list) or to info@nvaccess.org (although I am unsure of the latter). Regardless, since this is the Github issue tracker for NVDA, and this ticket is an enquiry rather than a bug report or feature suggestion, I suppose this can be safely closed. @leonardder @jcsteh
